public/third-party-cookies-check/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
I suggest we don't add this file until needed. It can be potentially in a completely different repo.

Reply:
yes it can. We decided (safe apps team) to keep it in the repo when the ticket was developed. Right now we have a subdomain pointing to the current site (https://gnosis-safe.io/app/third-party-cookies-check) that should be updated anyway when changing the main domain. Do you prefer to keep this in a separate repo? 

@dasanra

Reply:
We decided to keep it there because it was easier to find if you need to change/check after, as it's an "util" for the web itself. Changing it from repo will make it more difficult to find.

Maybe it can be removed from `public` and move it to the Safe Apps view folder, documenting the reason for this

Reply:
We will address this change at a later step